providers/trino/src/airflow/providers/trino/assets/trino.py

```diff
@@ -26,6 +26,10 @@
 def sanitize_uri(uri: SplitResult) -> SplitResult:
     if not uri.netloc:
         raise ValueError("URI format trino:// must contain a host")
+    try:
+        _ = uri.port
+    except ValueError:
+        raise ValueError("URI format trino:// must contain a valid port")
     if uri.port is None:
```

Comment:
This doesn't look related?  Was this change intentional?

Reply:
I'm not sure we need this. We probably can make that a separate test case instead of using paramtereize?

Reply:
modify done